### ISSUE TITLE
[Super Mario Maker] NSMBU theme is completely broken

### ISSUE THREAD

**Issue by slashiee** [slashiee/cemu_graphic_packs#11]
![](http://i.imgur.com/OumJaa6.png)

**Comment by slashiee** [comment]
Could possibly be a framebuffer issue, here's the list of resolutions that I got:
```=============================
CemuViewportSkimmer by MHVuze
=============================
Depending on your CPU and the log size, the processing might take some time.

0, 0, 1280, 720
0, 0, 854, 480
0, 0, 427, 240
0, 0, 106, 60
0, 0, 640, 360
0, 0, 160, 90
0, 0, 256, 256
0, 0, 64, 64
0, 0, 128, 256
0, 0, 32, 64
0, 0, 128, 128
0, 0, 1024, 1088
0, 0, 32, 32
0, 0, 16, 16
0, 0, 8, 8
0, 0, 48, 48
0, 0, 24, 24
0, 0, 4, 4
0, 0, 2, 2
0, 0, 768, 384
0, 0, 384, 192

Found 21 unique viewport resolutions and a total of 176471.
```

**Comment by slashiee** [comment]
Closed since Mario Maker support was removed e3f015c
